MovieMVPTests/ImageServiceTests.swift

```diff
@@ -0,0 +1,38 @@
+// ImageServiceTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+@testable import MovieMVP
+
+/// Тестирование имедж сервиса
+final class ImageServiceTests: XCTestCase {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let mockURLText = "/yDtgeQ2bwALLw1G2345XVeeP3wX.jpg"
```

Comment:
```suggestion
        static let mockURLText = ""
```
если этой картинки не будет, то тест провален. Но этот фильм (картинка)  может удалиться из сети и из памяти. Я сам так делаю, но видимо это тест ради теста

Reply:
согласен, но предполагаю что на работе у нас будет поднята своя картинка/ресурс на собственном сервере в которой мы будем точно уверены